### DISCUSSION TITLE
How to increase Icon Sizes of Control icons(Ex:- Pause, Play, Forward, Rewind etc.)?

### DISCUSSION THREAD

**Discussion by pavankumar-clasc** [jhomlala/betterplayer#754] [Q&A] [answered]
I want to increase the Icons size of the controls. I have tried increasing controlBarHeight. But, it doesn't make any changes. Suggest ways to increase the Icon size.

**Comment by jhomlala** [comment, marked as answer]
The easiest way is to create custom controls and to copy code from `better_player_cupertino_controls.dart` or `better_player_material_controls.dart`. Here's how to do that: https://github.com/jhomlala/betterplayer/tree/master/example/lib/pages/custom_controls
